language-plutus-core/exe/Main.hs

```diff
@@ -46,6 +52,8 @@ fileInput :: Parser Input
 fileInput = FileInput <$> strOption
   (  long "file"
   <> short 'f'
+  <> long "input"
+  <> short 'i'
```

Comment:
Two short options?

Reply:
I'd assumed this wouldn't work, but even if it does I think it's confusing and we shouldn't do it.

Reply:
I did this for backward compatibility.    It had `--file` for specifying the input file, but it would be a bit odd to have `-o` for output and `-f` for input so I added `-i`.  I'll get rid of `-f` and `--file` if no-one objects.

Reply:
Fine by me, but let's also ask @jmchapman.